src/module-1/card/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <div class="product-item">
+    <div class="product-img-wrapper">
+      <img class="product-img" src="../images/apple-watch.jpg" alt="product-1">
+    </div>
+    <div class="product-list">
+      <div>
+        <span class="product-rate">3.4
+          <img src="../images/star.svg" alt="">
```

Comment:
Please do not leave "alt" attribute empty. It specifies an alternate text for the image, if the image for some reason cannot be displayed. Also it useful for screen readers.